dist/svg.js

```diff
@@ -705,6 +705,63 @@ SVG.extend(SVG.PathArray, {
 
     return this
   }
+  // Test if the passed path array use the same commands as this path array
+, haveSameCommands: function(pathArray) {
```

Comment:
In case you wanna rename it: Something like `equalCommands`?
If not: I, at least, would name it `hasSameCommands`. Because its one patharray which `has` the same commands